ntoskrnl/ob/devicemap.c

```diff
@@ -536,9 +536,10 @@ ObInheritDeviceMap(IN PEPROCESS Parent,
 
 NTSTATUS
 NTAPI
-ObQueryDeviceMapInformation(IN PEPROCESS Process,
-                            IN PPROCESS_DEVICEMAP_INFORMATION DeviceMapInfo,
-                            IN ULONG Flags)
+ObQueryDeviceMapInformation(
+    _In_opt_ PEPROCESS Process,
+    _Out_ PPROCESS_DEVICEMAP_INFORMATION DeviceMapInfo,
+    _In_ ULONG Flags)
```

Comment:
Are you intending to introduce SAL annotations for the whole file or just that function?

Reply:
This function only, for the time being. (Noticed while checking #3634.)

Reply:
What's the point in annotating only one function in a file?

Reply:
It's a start... Does that cause any harm?